feather/base/src/metadata.rs

```diff
@@ -85,6 +85,30 @@ impl MetaEntry {
     }
 }
 
+pub enum Pose {
+    Standing,
+    FallFlying,
+    Sleeping,
+    Swimming,
+    SpinAttack,
+    Sneaking,
+    Dying,
+}
```

Comment:
Would this make sense to be in `libcraft`? Not saying it would but it might?

Reply:
There is a lot of code in base that should be part of libcraft, but that is beyond what this pull request is trying to do.

Reply:
That's not the point, you're adding new code. This new code should be in the most correct place. If you believe it would make sense to keep it there, leave it there. If it would be better placed in `libcraft` put it there.

Reply:
Pose is only used for metadata as far as I know so it makes sense to keep it with the rest of the metadata code. Just like how the impl ToMetaEntry is defined for all other types that could be expected. It could be argued that all of the metadata code could be in libcraft but the impl and definition of pose should be with the rest of it.

Reply:
It's annoying I can't resolve this comment